### DISCUSSION TITLE
null operator (of hoe dat ook heet, ipv 0/0)

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#61] [Ideas]
null operator (of hoe dat ook heet, ipv 0/0)

`Null[float32]`

Ook in een data array.

**Comment by basvanbemmel** [comment]
Is iets wat ik vaak gebruik 0/0 om null te forceren, ik zou er inderdaad syntax van maken. Maakt het ook eenduidiger; het lijkt nu meer een workaround/hack

**Comment by mtbeek32** [comment]
attribute&lt;uint32&gt; year  : [null]; works fine, so in a data array the null van already be used.

**Comment by jipclaassens** [comment]
Work-around could be:

```
container Units
{
        parameter<float32> Null_f := 0f/0fl;
}

attribute<float32> A := const(Units/Null_f,.);
```

**Comment by jipclaassens** [comment]
Afspraak, we maken een operator met de suffixes uit deze lijst: https://github.com/ObjectVision/GeoDMS/wiki/Value-type

null_f of null_b
